docs/sources/panels/visualizations/node-graph.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Node graph"
+keywords = ["grafana", "dashboard", "documentation", "panels", "node graph", "directed graph"]
+weight = 850
++++
+
+# Node graph panel
+
+> **Note:** This panel is currently in beta. Expect changes in future releases.
+
+The _Node graph_ can visualize directed graphs or networks. It uses directed force layout to effectively position the nodes so it can help with displaying complex infrastructure maps, hierarchies or execution diagrams.
+
+<img class="screenshot" src="/img/docs/v74/node-graph.png">
+
+## Data requirements
+
+The Node graph panel requires specific shape of the data to be able to display it's nodes and edges. This means not every data source or query can be visualized in this panel. If you want to use this as a data source developer see the section about data API.  
+
+// TODO: image or gif of node with context menu open
+
+The Node graph visualization consists of _nodes_ and _edges_.
+
+- A _node_ is DEFINE NODE IN THIS CONTEXT.
+- An _edge_ is DEFINE EDGE IN THIS CONTEXT.
```

Comment:
@oddlittlebird I am bit unsure about what sort of definition should be here. Feel like Edge and Node are basic english words and in this context they can represent different things, it will depend on the data source and what data it returns. So you cannot say that Node mean Application or Service because that won't be always true.

Reply:
Maybe something like:

A _node_ is displayed as a circle. A node might represent an application, a service, or anything else in a data network that you want to monitor.

An _edge_ is displayed as a line that connects two nodes. The connection might be a THING or OTHER THING, some relationship between the two nodes.